share/spice/get_events/get_events.js

```diff
@@ -0,0 +1,78 @@
+(function (env) {
+    "use strict";
+    
+    
+    var months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
+    // private functions
+    function getDate(date) {
```

Comment:
We now have Moment.js available to devs. You can likely remove most of this date code if you use moment :)

You'll need to wrap the `Spice.add()` call like so:

```js
DDG.require('moment.js' function(){
    Spice.add({
        // now you can use moment()
    });
});
```
```